### PR TITLE
CORE-334 Disable VICE launch button from opening Data window

### PR DESCRIPTION
**Pull request by ash3rz**

### CODE REVIEW

**Comment by ash3rz** [comment]
I'm gonna close this - I realized there are more icons we have in the Analysis window that probably should also not open the Data window when clicked, but I need to test them out.

**Comment by ash3rz** [comment]
It seems the other icons in the Analysis window don't trigger the Data window to open.  Reopening this PR.

File: react-components/src/analysis/view/AnalysesView.js [hunk 1]
```diff
@@ -92,20 +92,22 @@ function AnalysisName(props) {
...
-                onClick={() => handleGoToOutputFolder(analysis)}
-            >
+            <Fragment>
                 <LaunchIcon
                     onClick={() =>
                         handleInteractiveUrlClick(interactiveUrls[0])
                     }
                     id={build(baseId, ids.ICONS.INTERACTIVE)}
-                    style={{ color: palette.darkBlue }}
+                    className={interactiveStyle}
```
Comment (psarando) [inline react-components/src/analysis/view/AnalysesView.js]: ✨

**Comment by ash3rz** [comment]
Thanks for the review!

**Pull request merged**